### ISSUE TITLE
Deploy 1.23.0

### ISSUE THREAD

**Issue by Levdbas** [timber/timber#2820]
We have to wait for the docs deployment before we can start the release process. @gchtr Will be in charge of that. After that we can triple check the links in the merged changes to make sure the links inside the plugin notice will point to the right page.

~~Not sure if we want to label this as a path or minor version.~~ 1.23.0 it is!

## readme.txt
- [x] Update changelog info
- [x] Update "Test up to" version
- [x] Update Stable tag

## GitHub
- [x] Publish Release

## PHP files
- [x] Update version in `lib/Timber.php`
- [x] Update version in `bin/Timber.php`

## Deploy to WordPress.org
- [x] Run `./bin/deploy-to-wp-org 1.23.0`
- [x] Publish new tag for version
- [x] Update `trunk`

## Twitter
- [x] Tweet it out!
```It's a big release day! Version 1.23 is now live and marks the last Timber release as a plugin.```

**Comment by jarednova** [comment]
Thanks @Levdbas ! I think that it's worth making this a "major" version (1.23.0) so we recognize that as the final "deprecated" 1.x version. @gchtr can you give a nudge here once the docs deployment is complete? Then I can spin this out

**Comment by gchtr** [comment]
@jarednova @Levdbas The docs are updated. The new Guide is live: https://timber.github.io/docs/getting-started/switch-to-composer/.

**Comment by Levdbas** [comment]
Thank you @gchtr , I just checked all recent pr's that pointed to the guide and found one wrong link in a issue template. New PR @ https://github.com/timber/timber/pull/2823

**Comment by jarednova** [comment]
Wow, will NOT be missing SVN. It is done!
